### ISSUE TITLE
Missing the target side in the alignment group

### ISSUE THREAD

**Issue by monzug** [alpheios-project/alignment-editor-new#141]
Related to #134 
all suddenly I am not seeing anymore the right side in the alignment group. Note that I am uploading a single csv file for target and origin. Tokenization issues? I am not getting any error in the console. It does work if I am uploading a json file.

![Screen Shot 2021-02-01 at 4 19 31 PM](https://user-images.githubusercontent.com/41396793/106478252-4da76a00-64a9-11eb-8553-aaa4885ced0b.png)

**Comment by monzug** [comment]
OK. after several tries, I found out that it's the standard editor (in which way I don't know). I used a clean environment (Chrome/PC), did a couple of simple alignment that worked, download a json, upload and switch to standard theme, the target alignment is not visible any longer.

![right-alignment](https://user-images.githubusercontent.com/41396793/106504567-1d55d080-6495-11eb-951e-db51e18aee03.JPG)

**Comment by irina060981** [comment]
@monzug , I was not able to reproduce - but I believe that it is some bug after my last optimization - to render an editor only after first show (to reduce an amount to be rendered at once).
May be there are some delays here.
Will remove this optimization for now.

**Comment by monzug** [comment]
let me know when you have removed the optimization, then I'll try again

**Comment by irina060981** [comment]
@monzug, It is already removed - forgot to notice you, I am sorry :)

**Comment by irina060981** [comment]
Merged and closed

**Comment by monzug** [comment]
yes, this works now. I was also getting other errors like the following

**Comment by irina060981** [comment]
This error seems that the tokenizer was not available yesterday at that time.
And then it was fixed somehow.
Not by me :)
I believe that @balmas could know mor about this.

**Comment by balmas** [comment]
hmm. the tokenizer was only down briefly when I updated it and I don't think that was the same time Monica was testing.

The error makes me think that there is at least some bug in the code though, because if you see the URL it has `(tokenizationGroup.alpheios.getConfig)` in the path - this looks to me like there is a reference to the getConfig method as a string.  @irina060981 can you double check? Thanks!

**Comment by monzug** [comment]
I have to say that I haven't seen this error lately

**Comment by irina060981** [comment]
>The error makes me think that there is at least some bug in the code though, because if you see the URL it has (tokenizationGroup.alpheios.getConfig) in the path - this looks to me like there is a reference to the getConfig method as a string. @irina060981 can you double check? Thanks!

@balmas , this error says - that the fail was on the step - getConfig, when application starts to upload configuration (tokenization parameters) from the remote tokenizer to use them in text single blocks. So this text of the error is predefined.

```
class AdapterError extends Error {
  constructor (category, adapterName, methodName, messageError, statusCode) {
...
    if (this.adapter && this.methodName) {
      this.message = `${this.message} (${this.adapter}.${this.methodName})`
    }
  ...
  }
  ```
  
  This error is from the basic adapter on `window.fetch` fail.

**Comment by balmas** [comment]
ah ok. I think maybe in that case it would be better to show an error that is more meaningful to the user (e.g. "tokenization service unavailable") if possible.

**Comment by monzug** [comment]
I will reassign to @irina060981

**Comment by irina060981** [comment]
>ah ok. I think maybe in that case it would be better to show an error that is more meaningful to the user (e.g. "tokenization service unavailable") if possible.

It would be a change inside alpheios-core - to the client adapters

**Comment by irina060981** [comment]
>I will assign to @irina060981 a new issue with the above request from Bridget (#151)

As we have a separate issue for this - I will close the issue
